org.freeciv.sdl2.yml

```
@@ -0,0 +1,48 @@
+app-id: org.freeciv.sdl2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: freeciv-sdl2
+rename-icon: freeciv-client
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=~/.freeciv:create
+modules:
+  - name: sdl2-gfx
```

```suggestion
  - name: sdl2-gfx
  - build-options:
      - arch:
          - aarch64:
              - config-opts:
                  - --disable-mmx
```

Something like that.

---

might have been off on the yaml:

```
(flatpak-builder:2): Json-WARNING **: 23:01:09.563: Failed to deserialize "build-options" property of type "BuilderOptions" for an object of type "BuilderModule"
```